applications/asset_tracker/src/main.c

```diff
@@ -1790,8 +1790,8 @@ static void ui_evt_handler(struct ui_evt evt)
 
 void handle_nrf_modem_lib_init_ret(void)
 {
-#if defined(CONFIG_NRF_MODEM_LIB)
-	int ret = nrf_modem_lib_get_init_ret();
+#if defined(CONFIG_NRF_MODEM_LIB) && !defined(CONFIG_NRF_CLOUD_FOTA)
+	int ret = bsdlib_get_init_ret();
```

Comment:
This has now changed to `nrf_modem_lib_get_init_ret()`

Reply:
Too bad it wasn't caught by the CI... Is it ok to build asset_tracker w/o `NRF_CLOUD_FOTA` enabled? If so, perhaps we should add such a variant to the `sample.yaml`?

Reply:
yes it is ok to build w/o fota.  
by default fota is enabled when nrf_cloud is enabled.